models/engine/file_storage.py

```diff
@@ -68,3 +68,12 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """get the obejects by id, using dict's get()"""
+        if cls in classes and id and type(id) == str:
```

Comment:
`if <var> in <dict>` checks its keys, but if we are to match the use of `test_get_count.py` and not the task description, we want to check `if cls in classes.values()`